pytest_splunk_addon/standard_lib/addon_parser/fields.py

```diff
@@ -51,6 +51,15 @@ def __init__(self, field_json=None):
     def __str__(self):
         return str(self.name)
 
+    def __eq__(self, other: "Field"):
+        return self.__dict__ == other.__dict__
+
+    def __lt__(self, other: "Field"):
```

Comment:
Cannot see __gt__ in https://github.com/splunk/pytest-splunk-addon/blob/main/pytest_splunk_addon/standard_lib/addon_parser/fields.py
Probably worth to add, event if not needed now - just to have this part covered

Reply:
Tests seem to be passing without this thing, if we need it in the future, I would like to add it in the future.